examples/custom_checkers/r9900_redundant_condition.py

```diff
@@ -0,0 +1,16 @@
+def return_large_number(x: int) -> int:
+    """
+    Return number x only if it's greater than 1000
+
+    Preconditions:
+        - x > 1000
+    """
+    if x > 1000:
```

Comment:
Leave a comment indicating the error is on this line, and for all examples.

Follow the style in the existing files.